pkg/cmd/server/api/types.go

```diff
@@ -279,6 +279,21 @@ type MasterConfig struct {
 
 	// NetworkConfig to be passed to the compiled in network plugin
 	NetworkConfig MasterNetworkConfig
+
+	// JenkinsPipelineConfig holds information about the default Jenkins template
+	// used for JenkinsPipeline build strategy.
+	JenkinsPipelineConfig JenkinsPipelineConfig `json:"jenkinsPipelineConfig"`
```

Comment:
There has to be a better interface for github somewhere... This kind of thing should be possible to turn into todos on a PR.